lib/Simulation.thy

```diff
@@ -51,7 +51,7 @@ definition
 
 text \<open>
   The sequence of operations in the data type is then executed
-  in an initial state by initialising the private state, executing
+  in an initial state by initialising the observable state, executing
```

Comment:
It has to do both. Initially we only had trivial observable state, so the private state was the important bit, but I would vote for just saying "initialising the state".